### ISSUE TITLE
Cant sign in with beef?

### ISSUE THREAD

**Issue by EzScripts** [stormshadow07/BeeF-Over-Wan#54]
trying to sign in wiith beef and it says my username and password is beef and beef, that doesnt work and reloads the page? tips?

**Comment by ahmishra** [comment]
https://github.com/beefproject/beef/wiki/FAQ#how-do-i-configure-beef-with-ngrok, does this work for you? @EzScripts

**Comment by perez-afk** [comment]
Same error.

**Comment by xavitc100** [comment]
> https://github.com/beefproject/beef/wiki/FAQ#how-do-i-configure-beef-with-ngrok, does this work for you? @EzScripts

 public: 
            host: "<your-id>.ngrok.io"      # public hostname/IP address 
            port: "443"
            https: true

We just have to put our string generated by ngrok in "host: <your-id>"? I'm not sure if I understood it, pls can you confirm it? thank you

**Comment by xavitc100** [comment]
No, this doesn't work, when you write the correct credentials it reloads the page

**Comment by Astumblief** [comment]
> No, this doesn't work, when you write the correct credentials it reloads the page

Do you have a fix for that? I have the same problem.
